### ISSUE TITLE
Copyright in footer is missing fallbacks for Open Sans font.

### ISSUE THREAD

**Issue by supersam654** [daattali/beautiful-jekyll#215]
In `css/main.css`, "Open Sans" is used in several places with several fallbacks:

     font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;

However, one CSS selector is missing the fallbacks and should probably have them:

```
footer .copyright {
  font-family: Open Sans;
...
```

See https://github.com/daattali/beautiful-jekyll/blob/master/css/main.css#L275 for the exact spot.

**Comment by daattali** [comment]
Good find! Would you mind submitting a pull request?

Did you notice this because you saw the font broken on a browser?

**Comment by supersam654** [comment]
Just added a (tiny) PR for it.

I actually found it in a very roundabout way. I'm using [beautifulhugo](https://github.com/halogenica/beautifulhugo) which is a port of beautiful-jekyll in Hugo. I was pulling out the web fonts because I felt like it and noticed that the copyright in the footer looked really ugly but everything else just looked a little different. I did a little digging and found the line in the CSS file that was missing fallback fonts and did a bit more digging to discover that the file was pretty much untouched and came directly from here.

But no, the unmodified port of this theme wasn't broken in any browser I saw.
